moped-editor/src/views/projects/projectView/ProjectComponents.js

```diff
@@ -220,64 +197,73 @@ const ProjectComponents = () => {
             {componentsAvailable && (
               <ClickAwayListener onClickAway={handleComponentClickAway}>
                 <List className={classes.root}>
-                  {data.moped_proj_components.map((component, compIndex) => {
```

Comment:
this diff block renders oddly, but this is all moving to a `selectedProjectComponent` feature prop instead of merely using `projComponentId`.